.github/workflows/ci.yml

```diff
@@ -25,6 +25,7 @@ jobs:
     strategy:
       matrix:
         python-version: [3.8]
+        myst_nb: [0.13.1, 0.16.0]
```

Comment:
@AakashGfude should we add in all major versions for the test matrix?

Reply:
perhaps the largest minor within each major release.

Reply:
I guess we can add it @mmcky , but `0.14.0` had the biggest change with the code-cell containers and such. I think having one of `['0.14.0', '0.15.0', '0.16.0']` will be enough? Else it can increase the test time? I agree with having `0.13.2` instead of `0.13.1` though.

Reply:
OK. If it works across those other version I guess that's OK to have [`0.13.2` and `0.16` (or latest)]